src/Microsoft.DotNet.Darc/src/DarcLib/Actions/Remote.cs

```diff
@@ -1069,5 +1085,27 @@ private GitFile UpdateDotnetVersionGlobalJson(SemanticVersion incomingDotnetVers
             }
 
         }
+
+        private async Task<IEnumerable<DependencyDetail>> AddAssetLocationToDependenciesAsync(IEnumerable<DependencyDetail> dependencies)
+        {
+            foreach (var dependency in dependencies)
+            {
+                var matchingAssets = await GetAssetsAsync(dependency.Name, dependency.Version);
+
+                // Always look at the 'latest' asset to get the right asset even in stable build scenarios
```

Comment:
Would this break somehow if not all assets are published to same locations?

Reply:
How do you imagine this breaking?